### PR TITLE
Updated commands for consistency

### PR DESCRIPTION
**Pull request by VidyasagarMSC**

### CODE REVIEW

**Comment by VidyasagarMSC** [comment]
@beemarie Can you review and merge?

File: workshop/exercise-3/README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ What hostname should we use? Luckily for us, IBM Kubernetes Service gave us an e
 1. First, let's get the ingress subdomain for our cluster.
 
 	```
-	ibmcloud ks cluster-get <my-cluster-name>
+	ibmcloud cs cluster-get <my-cluster-name>
```
Comment (duglin) [inline workshop/exercise-3/README.md]: I think 'cs' is the old plugin

Comment (VidyasagarMSC) [reply workshop/exercise-3/README.md]: The latest update has support for both cs and ks.
> ibmcloud plugin show container-service                                                                                                    
> 
> Plugin Name                              container-service/kubernetes-service
> Plugin Version                           0.2.19
> Plugin SDK Version                       0.2.0
> Minimal IBM Cloud CLI version required   0.9.0

Comment (beemarie) [reply workshop/exercise-3/README.md]: We should use the new one -- i created a PR to change all cs -> ks, so you can just leave this line as ks.

**Pull request closed without merging**